ConorMFinn/CO600-SeriousCyberSec
# Create a Main Menu Scene for the Game

**Description of work.**
Created a main menu, contains a how to play section and level select. 
Currently only level one is available and it simply loads the "Hall" scene.
Also imports TextMeshPro

**To test:**
1. Open the MainMenu scene and play it
2. Check all the buttons work properly
3. Check the scaling is correct 
4. Check the scene loads properly (note: you may need to add both the menu and the hall scenes in `File -> Build Settings -> Scenes in Build`

Code review required on `MainMenuController.cs`

Fixes #45
